.gitignore

```diff
@@ -35,4 +35,14 @@ website/vendor
 *.winfile eol=crlf
 
 # Compilation outputs
-/plugins/kms/assets/boundary-plugin*
+/plugins/kms/assets/darwin/amd64/boundary-plugin*
+/plugins/kms/assets/darwin/arm64/boundary-plugin*
+/plugins/kms/assets/linux/386/boundary-plugin*
+/plugins/kms/assets/linux/amd64/boundary-plugin*
+/plugins/kms/assets/linux/arm/boundary-plugin*
+/plugins/kms/assets/linux/arm64/boundary-plugin*
+/plugins/kms/assets/windows/386/boundary-plugin*
+/plugins/kms/assets/windows/amd64/boundary-plugin*
+/plugins/kms/assets/freebsd/386/boundary-plugin*
+/plugins/kms/assets/freebsd/amd64/boundary-plugin*
+/plugins/kms/assets/freebsd/arm/boundary-plugin*
```

Comment:
Haven't tried it, but does `/plugins/kms/assets/*/*/boundary-plugin*` work? Not sure how greedy the globbing is in gitignores.